master/internal/api_model.go

```diff
@@ -38,14 +52,28 @@ func (a *apiServer) GetModels(
 		return nil, err
 	}
 
+	allowedUsers := strings.Join(req.users, ",")
```

Comment:
Huh - am I missing where this gets used? It looks an awful lot like part of one of the contains() approaches that is more concise but less scalable.

Reply:
Nope - this was replaced by `contains()` but I forgot to delete it.